app/graphql/types/ansible_variable_override.rb

```diff
@@ -0,0 +1,10 @@
+module Types
+  class AnsibleVariableOverride < GraphQL::Types::Relay::BaseObject
+    description 'Override value for Ansible Variable'
+
+    field :value, ::Types::RawJson, :null => false
+    field :element, ::Types::RawJson, :null => false
+    field :element_name, ::Types::RawJson, :null => false
```

Comment:
These names are a bit confusing, should we rename them to something more descriptive such as `override_source` or `overriding_resource`?

Reply:
These come directly from classification, are we sure we want to diverge on naming?